lib/src/modules/mod.rs

```diff
@@ -244,16 +251,17 @@ pub mod mods {
     /// # Example
     /// ```rust
     /// # use yara_x;
-    /// # let data = &[];
-    /// let modules_output = yara_x::mods::invoke_all(data);
+    /// let modules_output = yara_x::mods::invoke_all(&[], None);
     /// ```
-    pub fn invoke_all(data: &[u8]) -> Box<Modules> {
+    pub fn invoke_all(data: &[u8], meta: Option<&[u8]>) -> Box<Modules> {
```

Comment:
This is tricky, all modules are invoked with the same metadata and that doesn't make sense. I would leave `invoke_all` untouched for now. None of the invoked modules expect metadata anyways.

Reply:
youre right, this does not make sense the moment we add the possibility for each module to have its metadata

removed in [cca5adf](https://github.com/VirusTotal/yara-x/pull/183/commits/cca5adfaeb3671482a47f5b0a1a8e3f7d2a70fe8)